### ISSUE TITLE
Unsure how to use this extension properly

### ISSUE THREAD

**Issue by librarywebchic** [pavlakis/slim-behat-extension#1]
Hi,

I thought this extension would allow me to test my Slim app without starting web server. However, I get an error message (Failed to connect to localhost port 9090) when I run my tests.  Am I misinterpreting what this extension does?

**Comment by librarywebchic** [comment]
I don't think this extension works at all. Looking at the Laravel Extension this is based on I figured out how my behat.yml should be structured. However, when I did this I started getting the following message in my tests

"Argument 1 passed to Symfony\Component\HttpKernel\Client::__construct() must be an instance of Symfony\Component\HttpKernel\HttpKernelInterface, instance of Slim\App given" Looking at it I don't think this will work unless Slim supports this interface which it doesn't.
